pkg/disk/disk.go

```diff
@@ -0,0 +1,137 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package disk manages the persistent disk used to save containerd user data
+package disk
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+
+	"github.com/spf13/afero"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/path"
+)
+
+const (
+	diskName = "finch"
```

Comment:
Maybe add a comment in both here and yaml file to say the disk name must be same in the two places.